apache-activemq-mixin/config.libsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  _config+:: {
+    dashboardTags: ['apache-activemq-mixin'],
```

Comment:
since we said in the spec that this will support being run in kubernetes, we'll have to add a few things here. see the Aerospike mixin for an example (or Cassandra)